client/core/trade.go

```diff
@@ -992,7 +992,7 @@ func (c *Core) resendPendingRequests(t *trackedTrade) error {
 		if len(swapCoinID) != 0 && len(auth.InitSig) == 0 { // resend pending `init` request
 			err = c.finalizeSwapAction(t, match, swapCoinID, proof.Script)
 		} else if len(redeemCoinID) != 0 && len(auth.RedeemSig) == 0 { // resend pending `redeem` request
-			err = c.finalizeRedeemAction(t, match, swapCoinID)
+			err = c.finalizeRedeemAction(t, match, redeemCoinID)
```

Comment:
This was the bug.  It was sending an empty byte slice to the server and the server was rejecting the coin ID when trying to validate it with the wallet backend.  It was the wrong coin to send anyway.